.circleci/config.yml

```diff
@@ -292,6 +301,28 @@ jobs:
       - store_test_results:
           path: /home/frontend/feature
 
+  next_rails:
+    docker:
+      - <<: *frontend_base
+      - <<: *mariadb
+    steps:
+      - attach_workspace:
+         at: .
+      - run: *install_dependencies-next_rails
+      - run: *init_git_submodule
+      - run: *wait_for_database
+      - run: *create_test_db-next_rails
+      # TODO: Run all specs/tests with the next Rails version
```

Comment:
Not sure you should run all. This is costly, time wise.

Reply:
Yes, it's true. On the other end, how are we sure that we have no errors due to the next Rails version? I was thinking about running this only on branches which have a `next_rails:` prefix like `next_rails:fix-deprecation-warnings-123` or `next_rails:something-something`.

Reply:
Hm that sounds like a very good idea in combination with depfu.

Reply:
Filtering PRs based on their branch name isn't possible with CircleCI. I've looked at the environment variables like `CIRCLE_BRANCH` (it's always containing `pull/some_number` for PRs instead of the branch name given by the PR author). Filtering, with the `filters` key in the CircleCI configuration is only working for branches in the repository itself, not for PR since, just like `CIRCLE_BRANCH`, it's filtering PRs based on the value of `CIRCLE_BRANCH`. I think I'll give it a try with GItHub actions since it's possible to filter branches based on their branch name.